### PR TITLE
Add symbol layers as is on the mobile client

### PR DESCRIPTION
**Pull request by kahama94**
fix: #194  #180 #154 #187 
The issue depends on gisida-react v1.2.6

### CODE REVIEW

**Comment by moshthepitt** [comment]
> The issue depends on gisida-react v1.2.5 yet to be published

@kahama94 can we publish and update our dependencies in this PR?

**Comment by kahama94** [comment]
> can we publish and update our dependencies in this PR?

@moshthepitt most definitely.

**Comment by cKellyDesign** [comment]
@kahama94 - It looks like the Mosquito Collection layer is rendering both icons and circles, is this intentional? Looks like the circles can be removed if the icons are rendering, and maybe the icons size can increase a little when zooming in.

File: src/containers/pages/FocusInvestigation/map/active/index.tsx [hunk 1]
```diff
@@ -58,18 +61,26 @@ import plansReducer, {
...
+} from '../../../../../store/ducks/structures';
 import tasksReducer, {
   fetchTasks,
   getFCByPlanAndGoalAndJurisdiction,
-  getStructuresFCByJurisdictionId,
   reducerName as tasksReducerName,
   Task,
   TaskGeoJSON,
 } from '../../../../../store/ducks/tasks';
+import { jurisdictions } from '../../../../../store/ducks/tests/fixtures';
```
Comment (cKellyDesign) [inline src/containers/pages/FocusInvestigation/map/active/index.tsx]: @kahama94 - What's this being used for? Seems like fixtures should stick to tests?

File: src/helpers/utils.tsx [hunk 2]
```diff
@@ -109,13 +109,19 @@ export interface SiteConfigAppMapconfig {
...
+  imageUrl: string;
+}
 
 /** interface to describe Gisida site app configuration object */
 export interface SiteConfigApp {
   accessToken: string;
   apiAccessToken: string;
   appName: string;
   mapConfig: SiteConfigAppMapconfig;
+  mapIcons: MapIcons[];
```
Comment (cKellyDesign) [inline src/helpers/utils.tsx]: @kahama94 - Let's make `mapIcons` optional

**Pull request merged**